physics/GWD/drag_suite.F90

```diff
@@ -1414,6 +1414,1223 @@ subroutine drag_suite_run(                                           &
    return
    end subroutine drag_suite_run
 !-------------------------------------------------------------------
+
+   subroutine drag_suite_psl(                                           &
```

Comment:
It seems like drag_suite_run() was copied, updated and renamed drag_suite_psl()?
This is not the most CCPP way of doing things. drag_suite is a CCPP entrypoint that has CCPP _run() and _init() phases. drag_suite_psl is not visible to the CCPP!